go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/IBM/apigateway-go-sdk v0.0.0-20210714141226-a5d5d49caaca
 	github.com/IBM/appconfiguration-go-admin-sdk v0.3.0
 	github.com/IBM/appid-management-go-sdk v0.0.0-20210908164609-dd0e0eaf732f
-	github.com/IBM/cloud-databases-go-sdk v0.6.0
+	github.com/IBM/cloud-databases-go-sdk v0.7.0
```

Comment:
Has this sdk version released?

Reply:
Yep https://github.com/IBM/cloud-databases-go-sdk/releases/tag/v0.7.0